src/platform/packages/shared/response-ops/alerts-delete/api/last_run/use_alert_delete_last_run.test.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { renderHook, waitFor } from '@testing-library/react';
+import { useAlertDeleteLastRun } from './use_alert_delete_last_run';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { httpServiceMock } from '@kbn/core/public/mocks';
+import { getAlertDeleteLastRun } from './get_alert_delete_last_run';
+
+const http = httpServiceMock.createStartContract();
+
+jest.mock('./get_alert_delete_last_run', () => ({
+  getAlertDeleteLastRun: jest.fn(),
+}));
+
+describe('useAlertDeleteLastRun', () => {
```

Comment:
I think a test is missing for the scenario where the modal isOpen and **was not** closed before.

```
  if (isOpen) {
    const wasModalClosed = modalOpenCountRef.current === 0;

    
    if (wasModalClosed) { // <---------------------------- When this if fails.
      modalOpenCountRef.current = Date.now();
    }
  } else {
    ...
```

Reply:
done in 88ab0839d051552261b00b2e3bd9e10bdcbccafa